data/schema/schema.cfg

```diff
@@ -56,6 +56,10 @@
 		name="bool"
 		value="yes|no|true|false"
 	[/type]
+    [type]
```

Comment:
If you think it's a good idea, we could put the explanation of "can't override a super key with its absence" here? Just a thought, I don't strongly mind one way or the other.

Reply:
It is a great idea! Thanks! Added!